plugins/OAuth2/views/settings.php

```diff
@@ -0,0 +1,25 @@
+<h1><?php echo $this->data('Title'); ?></h1>
+
+<div class="PageInfo">
+    <h2><?php echo t('Create Single Sign On integration using the OAuth2 protocol!'); ?></h2>
```

Comment:
Don't think a bang is appropriate for basic instructions. This string needs to be put in locales, also. Cross-ref the PR.